cardinal/system/system_game_setup.go

```diff
@@ -124,19 +126,33 @@ func (s *GameSetup) setupBarn(world cardinal.WorldContext) {
 		enums.ObjectTypeDoor, enums.MaterialTypeWood,
 		"door", barnPlain, world)}
 
-	open2Forest := s.createAction(enums.ActionTypeOpen, " The window, glass and frame smashed"+
+	// KBARN -> E
+	open2Forge := s.createAction(enums.ActionTypeOpen, " The window, glass and frame smashed"+
 		" falls open.", false, false, false, 0, 0)
 
 	smashWindow := s.createAction(enums.ActionTypeBreak, " I love the sound of breaking glass"+
 		" especially when I'm lonely, the panes and the frame shatter"+
 		" satisfyingly spreading broken joy on the floor.",
-		true, false, false, open2Forest, 0)
+		true, false, false, open2Forge, 0)
 
-	windowActions := []uint32{open2Forest, smashWindow}
-	dObjs = append(dObjs, s.createDirObject(enums.DirectionTypeEast, enums.RoomTypePlain,
+	windowActions := []uint32{open2Forge, smashWindow}
+	dObjs = append(dObjs, s.createDirObject(enums.DirectionTypeEast, enums.RoomTypeForge,
 		enums.ObjectTypeWindow, enums.MaterialTypeWood,
 		"window", windowActions, world))
 
+	// KBARN -> DOWN
+	open2Cellar := s.createAction(enums.ActionTypeOpen, " The hay having been burned quickly, reveals the stairs cleared up.", false, false, false, 0, 0)
```

Comment:
"The hay having burnt fast reveals a set of stairs"

Reply:
done